free-programming-interactive-tutorials-en.md

```diff
@@ -147,6 +147,7 @@
 * [ABC of JavaScript : An Interactive JavaScript Tutorial](http://www.openjs.com/tutorials/basic_tutorial/)
 * [Codecademy jquery track](https://www.codecademy.com/learn/jquery)
 * [ES6 Interactive Guide](http://stack.formidable.com/es6-interactive-guide/#/)
+* [Freecodecamp:  Javascript algorithms and  data structures](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript) - Freecodecamp
```

Comment:
from https://www.freecodecamp.org/learn it seems this is labelled "Javascript algorithms and  data structures".
Don't invet your own title, please!

Reply:
Sir, The course's name is "Javascript Algorithms and Data Structures Certification"

Reply:
great, then use that title.